src/Bicep.Core/Analyzers/Linter/Rules/ParametersRequiredRule.cs

```diff
@@ -11,12 +11,12 @@ namespace Bicep.Core.Analyzers.Linter.Rules
 {
     public sealed class ParametersRequiredRule : LinterRuleBase
     {
-        public new const string Code = "Parameters Required";
+        public new const string Code = "parameters-required";
```

Comment:
suggest: params-required

Reply:
Oh, actually, Brian suggested we get rid of this rule completely.  It's no longer in the TTK.  I think I deleted it on my end.

Reply:
I will leave removal of rules up to you guys